### DISCUSSION TITLE
Merge history

### DISCUSSION THREAD

**Discussion by realyukii** [obsidianforensics/hindsight#178] [Q&A] [answered]
Does this tool capable for merge the browser's history from various device?

**Comment by RyanDFIR** [comment, marked as answer]
Yes, if you collect browser profiles from multiple different devices, put the profiles in a directory, then have Hindsight process that parent directory, it will process all the profiles together and merge the data into a single timeline.

Example file system:

```
evidence/
├─ system1_Default/
├─ system2_Default/
│  ├─ History
│  ├─ Web Data
│  ├─ Cookies
│  ├─ ...
├─ system2_Ryan/
```

And then the Hindsight command `hindsight.py -i evidence -o merged_report` will produce a `merged_report.xlsx` that has data from all three browser profiles in it.

**Reply by realyukii** [reply]
Thanks for the answers :)

and if you don't mind, could you please review my question on the [Google forum](https://support.google.com/chrome/thread/285654860?hl=en&sjid=1189069413721817502-AP)? You might have some insights that could help me there.
